### PR TITLE
Registration forms sub-chapter

### PR DESCRIPTION
**Pull request by EKaroune**
### Summary

Adding a separate sub-chapter fro registration forms.

Fixes #<NUM>

### List of changes proposed in this PR (pull-request)

- [x] add md files for main doc and template
- [x] add in toc.
- [x] check links etc

### What should a reviewer concentrate their feedback on?

- [x] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/collaboration/organising-conference/reg-forms.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Clearly state the next steps upon clicking the submit button.
+
+Link the registration form clearly on the webpage and advertise across different community platforms.
+
+(cl-organising-conferences-reg-forms-template)=
+## Template registration form
+
+We have made a template of all the different questions you might want to include on your registraion form. 
+
+You can see the template in our community handbook's tenplate collection.
```
Comment (EKaroune) [inline book/website/collaboration/organising-conference/reg-forms.md:71]:
```suggestion
You can see and used our registration form template in using this link 
```

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- [ ] Website
+- [ ] Newsletter
+- [ ] Twitter (add Twitter handle)
+- [ ] LinkedIn (add LinkedIn account link)
+- [ ] In another of our events 
+- [ ] Direct email
+- [ ] Email from a mailing list
+- [ ] Email from a colleague
+- [ ] I am staff member
+- [ ] Other [Please specify how you first heard about CW23.]
```
Comment (EKaroune) [inline book/website/collaboration/organising-conference/reg-form-template.md:140]:
```suggestion
- [ ] Other [Please specify how you first heard about the event.]
```

File: book/website/collaboration/organising-conference/reg-forms.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+- Questions about accessibility requirements can give answers that include health information.
+- Questions about dietary requirements can receive responses that record the religious belief of attendees and also health information such as allergies. 
+
+You must clearly explain why you’re asking for sensitive information and how you will handle this data.
+This includes clearly stating what you will do with the data afterwards - will this be to inform you on equality, diversity and inclusion, will you use it to write a report?
+
+Organisations usually have a privacy statement that can be used for this purpose. 
+An example of a privacy notice can be found on the The Alan Turing Institute website: https://www.turing.ac.uk/privacy-policy 
+
+(cl-organising-conferences-reg-forms=data-must)=
```
Comment (EKaroune) [inline book/website/collaboration/organising-conference/reg-forms.md:28]:
```suggestion
(cl-organising-conferences-reg-forms-data-must)=
```

**Comment by EKaroune** [comment]
@da5nsy and @BatoolMM - thanks for your help with this PR. 
I need your help! I am stuck on the numbering of the template. I want continuous numbering of the different questions and this is what is written on the md file but this renders in the book in a weird way. Any ideas of how to fix this? 
Thanks 😃

**Comment by da5nsy** [comment]
I'm away from a computer for the next couple of days, but I'd suggest playing around with a combination of escape characters and nesting the box lines with extra tabs

**Comment by EKaroune** [comment]
@BatoolMM - are you able to approve this for me please. Obviously take a quick look at the new chapters on reg forms and template of reg forms - thanks

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+```
+**Title:** *add title of event*
+
+**Description:** *add a short description of the event and include date, timings with timezone, location, online/in person only/hybrid.*
+
+*Example text*
+*We are convening a conference on ____ , which aims to bring together ________ to _______. This will include __________.*  
+
+*This event is invite only/by application/has a maximum of ___ attendees.  The event is in person/online/hyrbid at _____ on [add date here] between [add times] UTC.*
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-form-template.md:17]:
```suggestion
*This event is invite only/by application/has a maximum of ___ attendees.  The event is in person/online/hybrid at _____ on [add date here] between [add times] UTC.*
```

File: book/website/_toc.yml [hunk 5]
```diff
@@ -514,6 +514,11 @@ parts:
             file: collaboration/coworking
           - title: Organising Conferences
             file: collaboration/organising-conference
+            sections:
+              - title: Registration forms
+                file: collaboration/organising-conference/reg-forms
+              - title: Template for registration form
```
Comment (vhellon) [inline book/website/_toc.yml:520]:
```suggestion
              - title: Template for Registration form
```

File: book/website/_toc.yml [hunk 6]
```diff
@@ -514,6 +514,11 @@ parts:
             file: collaboration/coworking
           - title: Organising Conferences
             file: collaboration/organising-conference
+            sections:
+              - title: Registration forms
```
Comment (vhellon) [inline book/website/_toc.yml:518]:
```suggestion
              - title: Registration Forms
```

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 7]
```diff
@@ -0,0 +1,180 @@
+# Template for conference or event registration form
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-form-template.md:1]:
```suggestion
# Template for Conference or Event Registration Form
```

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+
+*This event is invite only/by application/has a maximum of ___ attendees.  The event is in person/online/hyrbid at _____ on [add date here] between [add times] UTC.*
+
+*We will be sending out more information about the content and focus of the event shortly.* 
+
+*Please register below to indicate that you will be attending.* 
+
+*Data privacy*
+
+*No personal data/personal data is collected from you by this form but if you want to read our privacy notice for events at _______, please find it here: [add link to privacy notice]*
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-form-template.md:25]:
```suggestion
*No personal data/personal data is collected from you by this form but if you want to read our privacy notice for events at _______, please find it here: [add link to privacy notice]* or *We collect personal data about individuals registering for this event. This data includes your name, your email address, your affiliation, country and job title. We will retain this registration information in our server for a year after the completion of the event.*
```

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+ - [ ] Yes
+ - [ ] No
+
+- Would you be willing to share your contact name and email with other conference attendees for networking purposes? (Select one) [compulsory question for all type of participants]
+ - [ ] Yes, I am comfortable sharing my contact information
+ - [ ] No, I prefer to keep my contact information private
+
+### Questions to collect attendees details for running the event (all information collected as open short answers)
+
+- Name (first name, family name)
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-form-template.md:48]:
```suggestion
- Name (first name, family name/surname)
```

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 10]
```diff
@@ -0,0 +1,180 @@
...
+
+- Title 
+
+- Preferred pronouns
+
+- Affiliation/Home institution/Organisation
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-form-template.md:54]:
```suggestion
- Affiliation/Home Institution/Organisation
```

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+
+- Career stage (Phases based on https://www.leru.org/files/UK-England.pdf)
+ - [ ] Phase 1 (for example: PhD candidate, Research/Teaching Assistant, Junior Research Software Engineer)
+ - [ ] Phase 2 (for example: Research Associate/Fellow, Lecturer, Research Software Engineer or Senior Research Software Engineer, Project Manager)
+ - [ ] Phase 3 (for example: Senior Lecturer, Reader, Senior Research Fellow, Research Group Leader, Research Software Group Leader, Chief Technical Officer, Programme Manager)
+ - [ ] Phase 4 (for example: Full Professor, Director of Research Computing, EU Project Coordinator, Director of Programme)
+
+- What is your research domain/field or area of work?
+
+- What is the closest JACS 3.0 Principal subject code (https://www.hesa.ac.uk/support/documentation/jacs) that clasifies your research area?
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-form-template.md:80]:
```suggestion
- What is the closest JACS 3.0 Principal subject code (https://www.hesa.ac.uk/support/documentation/jacs) that classifies your research area?
```

File: book/website/collaboration/organising-conference/reg-form-template.md [hunk 12]
```diff
@@ -0,0 +1,180 @@
...
+
+- Do you want to attend out conference dinner that will at [add venue] on [add date] at [add time]?
+ - [ ] Yes
+ - [ ] No
+
+- Would you like to apply for the conference childcare fund? *Provide details of the childcare fund including who can apply, how much and what details are needed to apply*
+ - [ ] Yes, I wish to apply for the childcare fund to support my attendance at the conference - please specify [text box]
+ - [ ] No
+
+- Would you like to apply for the conference’s accessibility fund? *Add details such as - The accessibility fund is designed to support attendees who need additional assistance to participate in the Conference. We can provide financial assistance (up to add amount/person) to help cover support costs for the duration of the conference, or for additional to support remote participation. Please select one of the following options: *
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-form-template.md:172]:
```suggestion
- Would you like to apply for the conference’s accessibility fund? *Add details such as - The accessibility fund is designed to support attendees who need additional assistance to participate in the conference. We can provide financial assistance (up to add amount/person) to help cover support costs for the duration of the conference, or for additional to support remote participation. Please select one of the following options: *
```

File: book/website/collaboration/organising-conference/reg-forms.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
+(cl-organising-conferences-reg-forms)=
+# Registration forms
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-forms.md:2]:
```suggestion
# Registration Forms
```

File: book/website/collaboration/organising-conference/reg-forms.md [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+
+(cl-organising-conferences-reg-forms-host)=
+## Where will you host your registration form?
```
Comment (vhellon) [inline book/website/collaboration/organising-conference/reg-forms.md:5]:
```suggestion
## Where will you host your Registration Form?
```

**Review by vhellon — COMMENTED** [review]
Have just corrected some typos and suggested some capitalisation (optional!) The code for the form looks great, although on the book preview it makes me scroll horizontally- not sure if you can make it so it fills the page?

**Review by BatoolMM — APPROVED** [review]

**Pull request merged**